scss/base/_variables.scss

```diff
@@ -28,6 +28,10 @@ $crm-grayblue-darker:       #c2cfd8 !default;
 $crm-white:                 #fff !default;
 $crm-black:                 #000 !default;
 
+// Font weights
+$crm-font-weight-heading:   600 !default;
```

Comment:
These variables are custom, so should not have `crm` prefix. Also they should be moved to the `// Custom` section.